src/index.js

```diff
@@ -38,7 +38,6 @@ import {
   VIEW_COURSE_PATH,
```

Comment:
This looks good and makes sense. Each view is wrapped in the dash in order to get all of the layout intact for each view. Awesomely done!